package/Dockerfile

```diff
@@ -1,3 +1,14 @@
-FROM alpine:3.14.2
+FROM registry.suse.com/bci/bci-micro:15.3.16.3
+
+ARG user=webhookreceiver
```

Comment:
I don't think that this should really be an arg - I don't see a reason for someone building to change this value and I'm thinking that it could really only cause issues down the line.

Reply:
I was thinking about that too. I only used `ARG` to not have to write the same value in multiple places. Do you think about a better way of doing this?

Reply:
You could use [ENV](https://docs.docker.com/engine/reference/builder/#env). Though it does appear that env is limited in terms of which instructions that it can be used in, so I'm not really sure that this is a substantial improvement.

That being said, near as I can tell what you are doing here is an accepted practice, so if we want to leave it this way I'm ok with it. I will mark this conversation as resolved, objection withdrawn.